router/diag_cntl.h

```diff
@@ -35,6 +35,23 @@
 #include "peripheral.h"
 #include "masks.h"
 
+struct diag_id_tbl_t {
+	struct list_head link;
+	uint8_t diag_id;
+	uint8_t pd_val;
+	uint32_t periph_id;
+	uint8_t pd_feature_mask;
+	char *process_name;
```

Comment:
I feel like to make our life easier, it would just be better to have process_name[MAX_DIAGID_STR_LEN]